<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_lista_producto</class>
 <widget class="QWidget" name="form_lista_producto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lista de productos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(140, 198, 63);</string>
  </property>
  <widget class="QLineEdit" name="lne_nombre_buscar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_limpiar">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>50</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Limpiar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lne_codigo_buscar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Producto Nombre :</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>651</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Producto Codigo :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_buscar">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_producto">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>661</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Codigo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nombre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Descripcion</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Stock</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cod producto</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btn_nuevo">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_actualizar">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Actualizar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
